README.md

```diff
@@ -1,63 +1,64 @@
 
-Using the latest `v1` API version of `ConsolePlugin` CRD, requires OpenShift 4.12
-and higher. For using old `v1alpha1` API version us OpenShift version 4.10 or 4.11.
+## Rough roadmap
 
-For an example of a plugin that works with OpenShift 4.11, see the `release-4.11` branch.
-For a plugin that works with OpenShift 4.10, see the `release-4.10` branch.
+* 2023 / v1: Adds a new **Dashboard** and **Metrics** tab to the Pipeline pages that shows aggregated PipelineRun stats from the [Tekton Results API](https://github.com/tektoncd/results)
+* 2024 / v2: Additional pages, like the Pipeline list and details pages are moved from the "static" [pipelines-plugin](https://github.com/openshift/console/tree/master/frontend/packages/pipelines-plugin) from the OpenShift Console into this project.
 
-[Node.js](https://nodejs.org/en/) and [yarn](https://yarnpkg.com) are required
-to build and run the example. To run OpenShift console in a container, either
-[Docker](https://www.docker.com) or [podman 3.2.0+](https://podman.io) and
-[oc](https://console.redhat.com/openshift/downloads) are required.
+## Compatiblity
 
-## Getting started
+This initial version of this plugin will be shipped with OpenShift Pipelines operator 1.1x and requires at least OpenShift 4.12.
 
-After cloning this repo, you should update the plugin metadata such as the
-plugin name in the `consolePlugin` declaration of [package.json](package.json).
+See also the OpenShift Pipelines [Compatibility and support matrix](https://docs.openshift.com/pipelines/latest/about/op-release-notes.html#compatibility-support-matrix_op-release-notes)
 
-```json
-"consolePlugin": {
-  "name": "console-plugin-template",
-  "version": "0.0.1",
-  "displayName": "My Plugin",
-  "description": "Enjoy this shiny, new console plugin!",
-  "exposedModules": {
-    "ExamplePage": "./components/ExamplePage"
-  },
-  "dependencies": {
-    "@console/pluginAPI": "*"
-  }
-}
-```
+## Development
 
-The template adds a single example page in the Home navigation section. The
-extension is declared in the [console-extensions.json](console-extensions.json)
-file and the React component is declared in
-[src/components/ExamplePage.tsx](src/components/ExamplePage.tsx).
+### Getting started
+
+Required tools:
+
+* [Node.js](https://nodejs.org/en/) v16 or newer and [yarn](https://yarnpkg.com) are required
+to build and run the example.
+* To run OpenShift console in a container, either
+[Docker](https://www.docker.com) or [podman 3.2.0+](https://podman.io) and
+[oc](https://console.redhat.com/openshift/downloads) are required.
 
 You can run the plugin using a local development environment or build an image
 to deploy it to a cluster.
 
-## Development
 
-### Option 1: Local
+### Option 1: Local development, with a local (cloned) console
+
+This option is a good candidate if you build and run the OpenShift Console already on your machine.
+
+In one terminal window, run:
+
+```
+cd console-plugin
+yarn install
+yarn start
```

Comment:
I have added a `dev` script to the `package.json`. 
This will restart the server on any code change. Can be mentioned here too :)
```suggestion
yarn start or yarn dev
```

Reply:
Add this to the README.

(Also when I think that what you have done might be good option for yarn start as well.)